homeassistant/components/evohome/__init__.py

```diff
@@ -399,14 +393,29 @@ def temperature_unit(self) -> str:
         """Return the temperature unit to use in the frontend UI."""
         return TEMP_CELSIUS
 
-    def _update_schedule(self) -> None:
+    async def _call_client_api(self, api_function) -> None:
+        try:
+            await api_function
+        except (aiohttp.ClientError, evohomeasync2.AuthenticationError) as err:
+            _handle_exception(err)
+
+        point_in_time = utcnow() + timedelta(seconds=2)
+
+        async_track_point_in_utc_time(
```

Comment:
We can use `helpers.event.async_call_later` instead.

Reply:
why? `async_call_later` is just a wrapper for `async_track_point_in_utc_time`?

Reply:
Yes. The point of the current code is to call a function after 2 seconds, right? Then we should use the best tool we have to do that.

Reply:
Done.